### PR TITLE
Added documentation for the schedule properties.

### PR DESCRIPTION
**Pull request by sagarkapare**

### CODE REVIEW

File: cdap-docs/developers-manual/source/building-blocks/schedules.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    ...
+    Map<String, String> scheduleProperties = Maps.newHashMap();
+    scheduleProperties.put("myProperty", "10");
+    scheduleProperties.put("anotherProperty", "anotherValue");
+
+    scheduleWorkflow("FiveHourSchedule", "0 */5 * * *", "MyWorkflow", scheduleProperties);
+    ...
+
+Every time ``FiveHourSchedule`` triggers, it passes the ``scheduleProperties`` as runtime arguments to the ``MyWorkflow``.
```
Comment (awholegunch) [inline cdap-docs/developers-manual/source/building-blocks/schedules.rst]: Every time the…

File: cdap-docs/developers-manual/source/building-blocks/schedules.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+The properties specified by the schedule can be overridden by the workflow's runtime arguments.
```
Comment (awholegunch) [inline cdap-docs/developers-manual/source/building-blocks/schedules.rst]: "can" or "will be"? Which has precedence? Perhaps:

The properties specified by the Schedule can be overridden by the Workflow's runtime arguments, with the Workflow's having precedence.

File: cdap-docs/developers-manual/source/building-blocks/schedules.rst [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _schedules:
+
+============================================
+Schedules
+============================================
+
+**Schedules** are used to execute :ref:`Workflows <workflows>` on a recurring basis. You can add a schedule (such as a
```
Comment (awholegunch) [inline cdap-docs/developers-manual/source/building-blocks/schedules.rst]: "Schedule" should start with a capital letter. \[common]

File: cdap-docs/developers-manual/source/building-blocks/schedules.rst (hunk 1 above)
Comment (sagarkapare) [inline cdap-docs/developers-manual/source/building-blocks/schedules.rst]: fixed.

File: cdap-docs/developers-manual/source/building-blocks/schedules.rst (hunk 2 above)
Comment (sagarkapare) [inline cdap-docs/developers-manual/source/building-blocks/schedules.rst]: Yes. Workflow's has precedence. Updated the line.

File: cdap-docs/developers-manual/source/building-blocks/schedules.rst (hunk 3 above)
Comment (sagarkapare) [inline cdap-docs/developers-manual/source/building-blocks/schedules.rst]: Changed.

**Comment by sagarkapare** [comment]
@awholegunch addressed the review comments. Please take a look again. Thanks!

**Comment by awholegunch** [comment]
LGTM.

**Pull request merged**